board/kontron/sa67/sa67.c

```diff
@@ -254,6 +254,11 @@ int board_late_init(void)
 #if IS_ENABLED(CONFIG_XPL_BUILD)
 void spl_perform_fixups(struct spl_image_info *spl_image)
 {
-	fixup_memory_node(spl_image);
+	if (IS_ENABLED(CONFIG_K3_DDRSS)) {
+		if (IS_ENABLED(CONFIG_K3_INLINE_ECC))
+			fixup_ddr_driver_for_ecc(spl_image);
+	} else {
```

Comment:
Nit: formattting:

if (IS_ENABLED(CONFIG_K3_DDRSS) && IS_ENABLED(CONFIG_K3_INLINE_ECC))
	fixup_ddr_driver_for_ecc(spl_image);
else
	fixup_memory_node(spl_image);

Reply:
That would actually change the logic. If `CONFIG_K3_DDRSS` is enabled but `CONFIG_K3_INLINE_ECC` is not, the upstream code does nothing, while your suggestion would call `fixup_memory_node(spl_image)`.